src/config/pragmas/indexes.js

```diff
@@ -9,6 +9,7 @@ module.exports = function configureIndexes ({ arc, errors }) {
   }
 
   let isCustomName = key => is.string(key) && key.toLowerCase() === 'name'
+  let isProjection = key => is.string(key) && key.toLowerCase() === 'project'
```

Comment:
Per https://github.com/architect/architect/issues/1083#issuecomment-852344502 changing this to `projection`